src/Raven.Server/Documents/BundleLoader.cs

```diff
@@ -40,61 +39,36 @@ public BundleLoader(DocumentDatabase database, ServerStore serverStore)
 
         public void HandleDatabaseRecordChange()
         {
-            VersioningStorage = VersioningStorage.LoadConfigurations(_database, _serverStore, VersioningStorage);
+            TransactionOperationContext context;
+            using (_serverStore.ContextPool.AllocateOperationContext(out context))
+            {
+                context.OpenReadTransaction();
+                var dbRecord = _serverStore.Cluster.ReadDatabase(context, _database.Name);
+                if (dbRecord == null)
+                    return;
+                VersioningStorage = VersioningStorage.LoadConfigurations(_database, dbRecord, VersioningStorage);
+                ExpiredDocumentsCleaner = ExpiredDocumentsCleaner.LoadConfigurations(_database, dbRecord, ExpiredDocumentsCleaner);
+                PeriodicExportRunner = PeriodicExportRunner.LoadConfigurations(_database, dbRecord, PeriodicExportRunner);
+            }
+                
         }
 
         /// <summary>
         /// Configure the database bundles if no changes has accord or when server start
         /// </summary>
         public void InitializeBundles()
```

Comment:
No, this is duplicating `HandleDatabaseRecordChange`, we should only call `HandleDatabaseRecordChange` to do all of this.